README.md

```diff
@@ -66,17 +66,19 @@ delete-variable --input <input.nc> --output <output.nc> --variable <varname>
 
 ### Shape file closing
 
-The scripts `close_boundary.py` and `generate_contour.sh` provide a way to close coastline contours when land intersects the data boundary. These files can be found in the `scripts/` subfolder of your local copy of this GitHub repository, so you should navigate there to use them (or replace the relative path in the commands below, e.g. `python close_boundary.py` -> `python ./scripts/close_boundary.py`).
+The script `close_boundary.py` provides a way to replace values along the domain boundary. This can be used to create "closed" shape files, that are useful for meshing, in combination with the GDAL command `gdal_contour`. The `close_boundary.py` file can be found in the `scripts/` subfolder of your local copy of this GitHub repository, so you should navigate there to use them (or replace the relative path in the commands below, e.g. `python close_boundary.py` -> `python ./scripts/close_boundary.py`).
```

Comment:
The _'closed' shape files_ are not only _useful for meshing_, they are _necessary for meshing_ in OceanMesh2D.